plugin-examples/pom.xml

```diff
@@ -1,205 +1,82 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
```

Comment:
I believe we removed the 'parent' here so the pom file can be copied to real plugins that will not declare themselves as 'children' of  styx-parent.   Do we definitely need that?